<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>526</width>
        <height>364</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="fovLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>FOV</string>
         </property>
         <property name="buddy">
          <cstring>fovSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="fovSpinBox">
         <property name="suffix">
          <string>Â°</string>
         </property>
         <property name="maximum">
          <number>180</number>
         </property>
         <property name="value">
          <number>120</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="focalPointLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Focal Point</string>
         </property>
         <property name="buddy">
          <cstring>focalPointSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QFrame" name="focalPointAuxFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="horizontalSpacing">
           <number>-1</number>
          </property>
          <property name="verticalSpacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item row="0" column="0" colspan="2">
           <widget class="QSpinBox" name="focalPointSpinBox">
            <property name="suffix">
             <string>%</string>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="imageWidthSizeLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&lt;width&gt;x&lt;height&gt;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="processedSizeLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Unwarped size is</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="interpolationLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Interpolation</string>
         </property>
         <property name="buddy">
          <cstring>interpolationComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="interpolationComboBox">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string comment="Image interpolation">Nearest Neighbor</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string comment="Image interpolation">Bilinear</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string comment="Image interpolation">Bicubic</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="finalSizeLabel">
         <property name="text">
          <string>Final Size</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QFrame" name="finalSizeFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QSpinBox" name="finalWidthSpinBox">
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="suffix">
             <string> px height</string>
            </property>
            <property name="maximum">
             <number>8000</number>
            </property>
            <property name="singleStep">
             <number>100</number>
            </property>
            <property name="value">
             <number>3000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="finalHeightSpinBox">
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="suffix">
             <string> px width</string>
            </property>
            <property name="maximum">
             <number>3000</number>
            </property>
            <property name="singleStep">
             <number>100</number>
            </property>
            <property name="value">
             <number>1000</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="skyUpCheckbox">
         <property name="text">
          <string>Sky is up?</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>91</width>
       <height>218</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>633</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>633</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fovSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>updateSizeLabel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>focalPointSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>updateSizeLabel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fovSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>updateFinalHeightRatio()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>finalWidthSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>updateFinalHeightRatio()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>finalHeightSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>updateFinalWidthRatio()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateSizeLabel()</slot>
  <slot>updateFinalHeightRatio()</slot>
  <slot>updateFinalWidthRatio()</slot>
 </slots>
</ui>
